README.md

```diff
@@ -1,13 +1,100 @@
-# Merlin Systems
+# [Merlin Systems](https://github.com/NVIDIA-Merlin/systems)
 
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/merlin-systems)
 [![PyPI version shields.io](https://img.shields.io/pypi/v/merlin-systems.svg)](https://pypi.python.org/pypi/merlin-systems/)
-[![Stability Alpha](https://img.shields.io/badge/stability-alpha-f4d03f.svg)](https://img.shields.io/badge/stability-alpha-f4d03f.svg)
-![GitHub Workflow Status](https://img.shields.io/github/workflow/status/NVIDIA-Merlin/systems/CPU%20CI)
 ![GitHub License](https://img.shields.io/github/license/NVIDIA-Merlin/systems)
+[![Documentation](https://img.shields.io/badge/documentation-blue.svg)](https://nvidia-merlin.github.io/systems/main/Introduction.html)
 
 Merlin Systems provides tools for combining recommendation models with other elements of production recommender systems like feature stores, nearest neighbor search, and exploration strategies into end-to-end recommendation pipelines that can be served with [Triton Inference Server](https://github.com/triton-inference-server/server).
 
+## Quickstart
+Merlin Systems uses the Merlin Operator DAG API, the same API used in [NVTabular](https://github.com/NVIDIA-Merlin/NVTabular) for feature engineering, to create serving ensembles. To combine a feature engineering workflow and a Tensorflow model into an inference pipeline: 
+
+```python
+import tensorflow as tf
+from nvtabular.workflow import Workflow
+from merlin.systems.dag import Ensemble, PredictTensorflow, TransformWorkflow
+
+# Load saved NVTabular workflow and TensorFlow model
+workflow = Workflow.load(nvtabular_workflow_path)
+model = tf.keras.models.load_model(tf_model_path)
+
+# Remove target/label columns from feature processing workflowk
+workflow = workflow.remove_inputs([<target_columns>])
+
+# Define ensemble pipeline
+pipeline = (
+	workflow.input_schema.column_names >> 
+	TransformWorkflow(workflow) >> 
+	PredictTensorflow(model)
+)
+
+# Export artifacts to disk
+ensemble = Ensemble(pipeline, workflow.input_schema)
+ensemble.export(export_path)
+```
+
+Once you have successfully exported your ensemble you can use the created directory to run an instance of tritonserver hosting your ensemble.
+```shell
+tritonserver --model-repository=/export_path/
+```
+
+See the [Getting Started with Merlin Systems](./examples/Getting_Started/Getting-started-with-Merlin-Systems.ipynb) example notebook for details. The notebook deploys this ensemble and demonstrates sending requests to Triton Inference Server.
+
+## Building a Four-Stage Recommender Pipeline
+
+Merlin Systems can also build more complex serving pipelines that integrate multiple models and external tools (like feature stores and nearest neighbor search):
```

Comment:
Maybe "...and external tools such as feature stores and nearest neighbor search. The following code sample shows the key classes and methods for integrating with a feature store:"